queries/water-z9.pgsql

```diff
@@ -35,6 +35,24 @@ FROM
         AND way_area::bigint > 102400 -- 4px
         AND mz_way12 && !bbox!
 
+    --
+    -- Water line geometries
+    --
+    UNION
+
+    SELECT name,
+           NULL AS area,
+           waterway AS kind,
+           'openstreetmap.org' AS source,
+           way AS __geometry__,
+           mz_id AS __id__
+
+    FROM planet_osm_line
+
+    WHERE
+        waterway IN ('canal', 'river')
```

Comment:
> Also, are there ever any waterways that we'd want to show that don't have a name set?

I'm not sure about this. I think that generally speaking that would certainly be smart way of reducing data size to only what matters, but I'm afraid about edge cases, specifically when a piece of a river for example would be missing the name, and so we would also miss the line (in the case we render the line geometries). So I'm balanced about this, and I think it's more of an editorial choice here, so I'll let you choose :)